job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext/deployToWeblogic.groovy

```diff
@@ -0,0 +1,43 @@
+job('example') {
+    publishers {
+        deployToWeblogic {
+            // these are the default values used for optional fields, when not overridden by closure
+            mustExitOnFailure false
+            forceStopOnFirstFailure false
+            deployingOnlyWhenUpdates false
+
+            deployedProjectsDependencies ''
+
+            // min. one task is required
+            task {
+                // required
+                weblogicEnvironmentTargetedName 'dev_environment'
+                // required
+                deploymentName 'myApplicationName'
+
+                deploymentTargets 'AdminServer'
+
+                // required
+                builtResourceRegexToDeploy 'myApp\\.ear'
+                // required
+                baseResourcesGeneratedDirectory ''
+                // required
+                taskName 'Deploy myApp to DEV Server'
+
+                jdkName 'JDK_7'
+                jdkHome 'C:\\Program Files\\Java\\jdk1.7.0_65'
+
+                // one of: WeblogicDeploymentStageModes
+                stageMode WeblogicDeploymentStageModes.BY_DEFAULT
+                commandLine '-debug -remote -verbose -name {wl.deployment_name} -targets {wl.targets} ' +
```

Comment:
Allow to accumulate fields which take long values, e.g.

```groovy
comandLine('-debug')
comandLine('-remote')
comandLine('-verbose')
```
The implementation should join the values with a space in between.